### PR TITLE
new HIP

### PR DESCRIPTION
**Pull request by KeithRettig**
I have no idea what I am doing, but here goes.

### CODE REVIEW

**Comment by KeithRettig** [comment]
I couldn't figure out how to put a table around the two examples.  If you could and are willing to do that, please do so.

**Comment by rorybokser** [comment]
Table formatting:

| Item                  | Rewards          |
| :-------------------- | ---------------: |
|DCs burned by SP       |       100,000,000|
|Price of DC            |          $0.00001|
|Price of MOBILE*       |           $0.0001|
|DCs burned in MOBILE** |        10,000,000|
|10% SP bucket          |500,000,000 MOBILE|
|SP rewards             | 10,000,000 MOBILE|

File: 0000-Proportional-Service-Provider Rewards [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+While approval of this approach is probably not necessary for Helium Mobile to start offloading data to the Helium Mobile Network, it does modify a policy that is important for Helium Mobile to function.
+
+## Implementation
+We leave the implementation of the smart contract components, verifiability, and Service Provider compliance up to the Helium Core Developers to determine.
+
+## Drawbacks
+There are no perceived drawbacks.
+
+## Unresolved Questions
+There are no unresolved questions.
```
Comment (vincenzospaghetti) [inline 0000-Proportional-Service-Provider Rewards:54]: There has to be at least one question. 
What about:

When will HIP 74, modeled coverage, be implemented? The answer to this question affects this HIP.

File: 0000-Proportional-Service-Provider Rewards [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# HIP XX: Proportional Service Provider Rewards
+- Author(s): @zer0tweets, @meowshka (Nova Labs, Inc.), @KeithRettig (Discord @KeithR)
+- Start Date: 2023-05-29
+- Category: Economic, Technical
+- Original HIP PR: #nnn
+- Tracking Issue: #nnn
+- Voting Requirements: veMOBILE Holders
+
+## Summary
+This HIP proposes to make modifications to how Service Providers are eligible for rewards from the Service Provider MOBILE Reward bucket.
```
Comment (vincenzospaghetti) [inline 0000-Proportional-Service-Provider Rewards:10]: This summary is a bit vague and lacks the core points of your argument:

Can you summarize the whole text in this part? Be sure to include the main points. Should look something like this and you can use this as a starting point:

The proposal suggests using a usage-based approach to calculate rewards for Service Providers in the Helium MOBILE Network, based on the amount of Data Credits burned. Service Providers would be rewarded in MOBILE tokens at a 1:1 ratio for the burned Data Credits, with proportional distribution if the total exceeds the reward bucket value.

File: 0000-Proportional-Service-Provider Rewards [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+## Motivation
+The primary motivation is to prevent a scenario in which a Service Provider is rewarded the full 10% MOBILE reward bucket (provided HIP79 passes) while burning little to no Data Credits.
```
Comment (vincenzospaghetti) [inline 0000-Proportional-Service-Provider Rewards:13]: Can you elaborate this section as well? This is a great place for context. Give a reader context about the mobile network, about the problems it is having, and the problem that sparked your desire to write this.

Side note: Your motivation is about gaming, right? I believe I've heard that as part of the working group's discussion. It would be good to add any info about how you all modeled gaming or expect gaming based on how gaming on the IOT network went to show how you identified the problem and how someone could game the network this way.

File: 0000-Proportional-Service-Provider Rewards [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+To illustrate, here are two scenarios:
+Value of DCs burned for data traffic is less than the Service Provider reward bucket (10% of MOBILE emissions).
+
+DCs burned by SP        100,000,000
+Price of DC             $0.00001
+Price of MOBILE*        $0.0001
+DCs burned in MOBILE**  10,000,000
+10% SP bucket           500,000,000 MOBILE
+SP rewards              10,000,000 MOBILE
+
```
Comment (vincenzospaghetti) [inline 0000-Proportional-Service-Provider Rewards:30]: Add markdown formatting for this as a table.

Try this tool: https://www.tablesgenerator.com/markdown_tables

Anything you put above the line will format as a column header. anything below is part of the rows.
```
`|   |   |   |   |   |
|---|---|---|---|---|
|   |   |   |   |   |`
```

File: 0000-Proportional-Service-Provider Rewards [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+Price of MOBILE*        $0.0001
+DCs burned in MOBILE**  10B
+10% SP bucket           500,000,000 MOBILE
+SP rewards              500,000,000 MOBILE
+
+
+* The price chosen is an example, not the actual price. The actual price will be determined by the Price Oracle on the Solana blockchain during the rewards calculation time.
+** Amount of MOBILE tokens with the same value as the value of the DCs burned by SP: (10,000 DC * $0.00001 (fixed price of DC) / $0.0001 (example price of MOBILE))
+
+While approval of this approach is probably not necessary for Helium Mobile to start offloading data to the Helium Mobile Network, it does modify a policy that is important for Helium Mobile to function.
```
Comment (vincenzospaghetti) [inline 0000-Proportional-Service-Provider Rewards:45]: Can you explain this further? Why is it important for Helium Mobile, the service provider, to function? Give the reader the context/background.

**Review by vincenzospaghetti — CHANGES_REQUESTED** [review]

**Comment by KeithRettig** [comment]
I made all of the edits requested but then merged the file up to my main branch.  This had the effect of making this copy of the file outdated and not the one that should be merged back up.  So I am closing this pull request so that the three HIPs in my more recent pull request will be the only one outstanding.
